README.md

```diff
@@ -731,6 +731,7 @@ Name | Website | Region
 [Xapo](/company-profiles/xapo.md) | https://xapo.com/en/ | Worldwide
 [XP Inc](/company-profiles/xp-inc.md) | https://www.xpi.com.br/ | Brazil
 [Yandex](/company-profiles/yandex.md) | https://yandex.com/ | Worldwide
+[Yassir](/company-profiles/yassir.md) | https://jobs.lever.co/Yassir | Algeria, Worldwide
```

Comment:
This link should be the link to the main site, not the job careers page

Reply:
done!